vis: 1
inverse_flag: 0
forward_distance: 75
end_x: 12.5